include/deal.II/grid/tria_description.h

```diff
@@ -398,6 +398,8 @@ namespace TriangulationDescription
      * Properties to be use in the construction of the triangulation.
      */
     Settings settings;
+
+    typename Triangulation<dim, spacedim>::MeshSmoothing smoothing;
```

Comment:
Can you add a comment?

Reply:
@bangerth I have not finished this PR, yet. See the description of this PR. Also, I would be interested if you and @luca-heltai agree on having in `TriangulationDescription::Description` the information regarding to mesh smoothing. `TriangulationDescription::Description` is in some form used by all triangulations now...

Reply:
If you include the settings into the structure then the smoothing flags make sense as well.

Reply:
`Settings` is actually what will be removed from the struct and moved to the constructor of `p:f:t` as you suggested in an other PR.

Reply:
OK, and I still agree with that. Then I would also move the flags along with the settings.